src/configuration.ts

```diff
@@ -670,7 +622,7 @@ export function resolveTeamIds(cfg: Configuration): string[] {
  * @return the resolved workspace IDs
  */
 export function resolveWorkspaceIds(cfg: Configuration): string[] {
-    const teamIds = resolveTeamIds(cfg);
+    const teamIds = [];
     if (process.env.ATOMIST_WORKSPACES) {
         cfg.workspaceIds = process.env.ATOMIST_WORKSPACES.split(",");
     } else if (config("workspaceIds")) {
```

Comment:
there's an if statement here that can go away, that checks the length on teamIds[]

Reply:
Good catch. Will remove.